joblib/memory.py

```diff
@@ -518,6 +521,25 @@ def __reduce__(self):
         return (self.__class__, (self.func, self.store_backend, self.ignore,
                 self.mmap_mode, self.compress, self._verbose))
 
+    def in_store(self, *args, **kwargs):
+        """"Return true if the store contains results for set of parameters.
+
+        Note that this will not check if results can be loaded, call the
+        function to achieve this.
+
+        Returns
+        -------
+            cached: bool
+                True if calling function with these arguments would
+                load from backend, else False.
+        """
+        func_id, args_id = self._get_output_identifiers(*args, **kwargs)
+        if (self._check_previous_func_code(stacklevel=4) and
```

Comment:
This could be done in a one-liner:
```python
return (self._check_previous_func_code(stacklevel=4) and self.store_backend.contains_item([func_id, args_id]))
```
(You will just have to split the line to be pep8 compliant)